packages/fiori/src/UploadCollectionItem.js

```diff
@@ -0,0 +1,380 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import Input from "@ui5/webcomponents/dist/Input.js";
+import Label from "@ui5/webcomponents/dist/Label.js";
+import Link from "@ui5/webcomponents/dist/Link.js";
+import ListItem from "@ui5/webcomponents/dist/ListItem.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import UploadState from "./types/UploadState.js";
+import "@ui5/webcomponents-icons/dist/icons/refresh.js";
+import "@ui5/webcomponents-icons/dist/icons/stop.js";
+import {
+	UPLOADCOLLECTIONITEM_CANCELBUTTON_TEXT,
+	UPLOADCOLLECTIONITEM_RENAMEBUTTON_TEXT,
+	UPLOADCOLLECTIONITEM_ERROR_STATE,
+	UPLOADCOLLECTIONITEM_UPLOADING_STATE,
+	UPLOADCOLLECTIONITEM_READY_STATE,
+	UPLOADCOLLECTIONITEM_RETRY_BUTTON_TEXT,
+	UPLOADCOLLECTIONITEM_TERMINATE_BUTTON_TEXT,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Template
+import UploadCollectionItemTemplate from "./generated/templates/UploadCollectionItemTemplate.lit.js";
+
+// Styles
+import UploadCollectionItemCss from "./generated/themes/UploadCollectionItem.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-upload-collection-item",
+	properties: /** @lends sap.ui.webcomponents.fiori.UploadCollectionItem.prototype */ {
+		/**
```

Comment:
make sure all the public properties have public, type and defaultvalue tags

Reply:
Done